index.html

```diff
@@ -909,6 +909,85 @@ <h4>
 setTimeout(() => controller.abort(), 3000);
     </pre>
   </aside>
+
+  <aside title="Read a external record with sub-records" class="example">
+      <p>
```

Comment:
I contest the usefulness of this vs the added complexity.
If storage efficiency is of concern, an external type with binary (e.g. CBOR) would be a yet more efficient solution, without the arbitrary conventions made down this example.

Reply:
It is supported by NDEF and even used in standard records like smart poster.

You can of course contest using NDEF at all (and write data directly to the tag using the type specific protocols), but hey that is what we are spec'ing.

Reply:
If I am reading correctly the NFC Forum Smart Poster spec, there is no word about "sub-record", only about records (of type Title, URI, Action, Icon, Size, Type). The term "sub-record" seems to be made up outside of NFC Forum, so it's an application level term from NFC point of view. Read my previous comment in that context: apps could as well choose a more efficient storage method inside the allowed NDEF records. If we expose an app-level mechanism on top of NDEF in the Web NFC spec that needs to go though the usual feature addition path: use case, issue, group discussion, PR.

Reply:
OK, it definitely embeds the records and that is also defined how to do in other NFC specs. Whether we call them sub records or embedded records, I don't care.